libcore/src/core.cpp

```diff
@@ -66,6 +66,7 @@ bool MCThrow(uint32_t p_error)
 #include <crtdbg.h>
 #include <dbghelp.h>
 
+// WIN64-CHECK
```

Comment:
They were just markers for things I needed to double-check - will do so and remove :)